src/js/index.js

```diff
@@ -0,0 +1,27 @@
+const submit = e => {
+  e.preventDefault();
+  let input = document.querySelector('input');
+  if(input.value != '') {
```

Comment:
try to use mosty strict comparison, unless intended
```suggestion
  if(input.value !== '') {
```